payment_paybox/controllers/__init__.py

```diff
@@ -0,0 +1,3 @@
+# flake8: noqa
```

Comment:
You don't need to exclude it from flake8. It's already handled on current Travis configuration